1-WebApp-storage-managed-identity/README.md

```diff
@@ -0,0 +1,221 @@
+- create a web app
+- publish the web app to Azure
+
+For information on how to do this from VS Code using the [App Service Extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice), see the [tutorial](https://docs.microsoft.com/azure/developer/javascript/tutorial/deploy-nodejs-azure-app-service-with-visual-studio-code?tabs=bash). Use the same resource group that you used in step 2 instead of creating a new resource group.
+
+After you've deployed the web app to Azure, [configure the Azure App Service authentication/authorization module](https://docs.microsoft.com/azure/app-service/scenario-secure-app-authentication-app-service).  Also verify that only users in your organization can access the web site.
+
+### Step 4: Configure the App Service to use your Azure storage account and blob container
+
+Now you need to add environment variables to the App Service where you deployed your web app.
+
+1. In the [Azure portal](https://portal.azure.com), search for and select **App Service**, and then select your app.
+1. Select **Configuration** blade on the left, then select **New Application Settings**.
+1. Add the following variables (key-value pairs):
+    1. **STORAGE_ACCOUNT_NAME**: the name of the storage account you created in step (2)
+    1. **BLOB_CONTAINER_NAME**: the name of the container blob you created in step (2)
+
+Wait for a few minutes for your changes on **App Service** to take effect.
+
+### Step 5: Enable managed identity on an app
+
+Read this article to learn how to [enable a managed identity on a web app](https://docs.microsoft.com/azure/app-service/scenario-secure-app-access-storage#enable-managed-identity-on-an-app).
+
+### Step 6: Grant access to the storage account
+
+You need to grant your web app access to the storage account before you can create, read, or delete blobs. In a previous step, you configured the web app running on App Service with a managed identity. Using Azure RBAC, you can give the managed identity access to another resource, just like any security principal. The Storage Blob Data Contributor role gives the web app (represented by the system-assigned managed identity) read, write, and delete access to the blob container and data. Read this article to learn how to [grant access to the storage account](https://docs.microsoft.com/azure/app-service/scenario-secure-app-access-storage#grant-access-to-the-storage-account).
+
+### Step 7: Visit the web app
+
+Open a browser and navigate to the deployed web app (replace *web-app-name* with the name of your web app): `https://web-app-name.azurewebsites.net/`
+
+## About the code
+
+This sample is built using the [@azure/identity](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/README.md) and [@azure-samples/msal-express-wrapper](https://github.com/Azure-Samples/msal-express-wrapper) packages for authentication and authorization.
+
+### Add authentication to your web app
```

Comment:
you might need to be more prescriptive here, like what needs adding to a web project to enable easyauth